doc/reference/storage_zfs.md

```diff
@@ -114,8 +114,8 @@ Key                           | Type                          | Default
 
 Key                     | Type      | Condition                 | Default                                        | Description
 :--                     | :---      | :--------                 | :------                                        | :----------
-`block.filesystem`      | string    | `zfs.block_mode` enabled  | same as `volume.block.filesystem`              | {{block_filesystem}}
-`block.mount_options`   | string    | `zfs.block_mode` enabled  | same as `volume.block.mount_options`           | Mount options for block-backed file system volumes
+`block.filesystem`      | string    | block-based volume with content type `filesystem` (`zfs.block_mode` enabled)  | same as `volume.block.filesystem`              | {{block_filesystem}}
+`block.mount_options`   | string    | block-based volume with content type `filesystem` (`zfs.block_mode` enabled) enabled  | same as `volume.block.mount_options`           | Mount options for block-backed file system volumes
```

Comment:
```suggestion
`block.mount_options`   | string    | block-based volume with content type `filesystem` (`zfs.block_mode` enabled) | same as `volume.block.mount_options`           | Mount options for block-backed file system volumes
```

Reply:
Oops, where did that come from? Thanks, fixed!